Source/ThirdParty/ANGLE/src/common/PoolAlloc.h

```diff
@@ -164,19 +128,15 @@ class PoolAllocator : angle::NonCopyable
     // any) will align to pointer size by extension (since mAlignment is made aligned to at least
     // pointer size).
     size_t mCurrentPageOffset;
-    // List of popped memory
-    PageHeader *mFreeList;
     // List of all memory currently being used.  The head of this list is where allocations are
     // currently being made from.
     PageHeader *mInUseList;
-    // Stack of where to allocate from, to partition pool
-    AllocStack mStack;
 
     int mNumCalls;       // just an interesting statistic
     size_t mTotalBytes;  // just an interesting statistic
 
 #else  // !defined(ANGLE_DISABLE_POOL_ALLOC)
-    std::vector<std::vector<void *>> mStack;
+    std::vector<void *> mStack;
```

Comment:
Nit: This is no longer really `mStack` but what ever each level in the stack contained. Is `mAllocs` more appropriate?